### DISCUSSION TITLE
how do you use this library with vite and react js?

### DISCUSSION THREAD

**Discussion by yossularko** [JonasKruckenberg/imagetools#467] [Q&A]
how to use this library for vite react js?

**Comment by JonasKruckenberg** [comment]
Follow [this guide here](https://github.com/JonasKruckenberg/imagetools/tree/main/packages/vite#usage) to add the plugin to your vite config and then you can use the directives in import statements:

```jsx
import image from "example.jpg?w=500&format=avif"

function App() {
   return <img src={image}/>
}
```
